packages/frames.js/src/utils.ts

```diff
@@ -81,3 +81,59 @@ export function isValidVersion(version: string): boolean {
 
   return true;
 }
+
+export function getEnumKeyByEnumValue<
+  TEnumKey extends string,
+  TEnumVal extends string | number,
+>(
+  enumDefinition: { [key in TEnumKey]: TEnumVal },
+  enumValue: TEnumVal
+): string {
+  return (
+    Object.keys(enumDefinition)[
+      Object.values(enumDefinition).indexOf(enumValue)
+    ] ?? ""
+  );
+}
+
+export function extractAddressFromJSONMessage(message: any): `0x${string}` {
+  const { data } = Message.fromJSON(message);
+
+  if (!data) {
+    throw new Error("Invalid message provided. Message data is missing");
+  }
+
+  if (data.type !== MessageType.VERIFICATION_ADD_ETH_ADDRESS) {
```

Comment:
so having any solana verifications will throw an error on the existing endpoints?

Reply:
not ideal we need to handle solana things

Reply:
I don't know what you are talking about :D This is doing the same thing as previosly in version `0.5.0`

Reply:
For now solana addresses are ignored.